chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -426,44 +431,64 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
     const confirmationPaneProps = initConfirmationPropsComposer(props);
 
     return (
-        <Composer
-            {...webChatProps}
-            styleOptions={webChatStyles}
-            directLine={props.webChatContainerProps?.directLine ?? adapter ?? defaultWebChatContainerStatefulProps.directLine}>
-            <Stack
-                id={widgetElementId}
-                styles={generalStyles}
-                className={props.styleProps?.className}>
+        <>
```

Comment:
Can we check if there is any option to customize the default scrollbar with fluent UI instead of adding style tag here?

Reply:
we want to allow customization for scrollbar instead of using default one.

Reply:
I meant is there a way to use Fluent UI for scrollbar customization
Seems like tit is not possible https://github.com/microsoft/fluentui/issues/8302